Mathlib/Data/Matrix/Hadamard.lean

```diff
@@ -148,19 +148,18 @@ variable [Fintype m] [Fintype n]
 
 variable (R) [Semiring α] [Semiring R] [Module R α]
 
-theorem sum_hadamard_eq : (∑ (i : m) (j : n), (A ⊙ B) i j) = trace (A ⬝ Bᵀ) :=
+theorem sum_hadamard_eq : (∑ i : m, ∑ j : n, (A ⊙ B) i j) = trace (A ⬝ Bᵀ) :=
```

Comment:
That's a bit sad that this notation no longer works

Reply:
I agree!